.env.dev

```diff
@@ -15,9 +15,9 @@ CANONICAL_HOST=localhost
 DEBUG_ERRORS=true
 FORCE_SSL=false
 PORT=4000
-SECRET_KEY_BASE= # Generate with `mix phx.gen.secret`
-SESSION_KEY= # Generate with `mix phx.gen.secret`
-SIGNING_SALT= # Generate with `mix phx.gen.secret`
+SECRET_KEY_BASE=__Generate_a_64_bytes_secret_with_phoenix_generator:_mix phx.gen.secret__
+SESSION_KEY=__Generate_a_session_key_with_mix phx.gen.secret__
+SIGNING_SALT=__Generate_a_signing_salt_with_mix phx.gen.secret__
```

Comment:
> Running the application with the vanilla docker-compose and .env.dev caused an endpoint error:

Yes, this is by design. We don’t want to hardcode secrets in .env.dev. We want developers using this boilerplate project to have to generate them.

Reply:
I will add instruction in the `Readme` then to explain this mandatory step for both development and testing the OTP.

Reply:
I will add instruction in the `Readme` then to explain this mandatory step for both development and testing the OTP.

Reply:
Since you make modification, can we remove the `mix phx.gen.secret` on `SESSION_KEY`? The session key is not meant to be stored as a secret key. It should be set as `elixir-boilerplate`. It will then be renamed to the app name on the setup script.